cmd/root.go

```diff
@@ -146,8 +146,29 @@ func initConfig() {
 		}
 	}
 
-	setLogging(log.ErrorLevel)
+	// Check if debug flag is set first, if not set then we look into config (default should be ErrorLevel)
 	if param.Debug.GetBool() {
 		setLogging(log.DebugLevel)
+	} else {
```

Comment:
Use the [ParseLevel(string)](https://github.com/sirupsen/logrus/blob/v1.9.3/logrus.go#L25) method instead... utilizes slightly different string constants but is better to have someone else do the heavy lifting.